.github/workflows/build-test.yaml

```diff
@@ -1099,33 +1109,34 @@ jobs:
           EXIT_CODE=$?
           if [ $EXIT_CODE -ne 0 ]; then
             echo "------pods:"
-            kubectl -n $APP_SLUG get pods
+            kubectl -n "$APP_SLUG" get pods
             echo "------kotsadm logs"
-            kubectl logs -l app=kotsadm --tail=100 --namespace $APP_SLUG
+            kubectl logs -l app=kotsadm --tail=100 --namespace "$APP_SLUG"
             exit $EXIT_CODE
           fi
 
           COUNTER=1
-          while [ "$(./bin/kots get apps --namespace $APP_SLUG | awk 'NR>1{print $3}')" != "$LATEST_APP_VERSION_LABEL" ]; do
-            COUNTER=$[$COUNTER +1]
+          while [ "$(./bin/kots get apps --namespace "$APP_SLUG" | awk 'NR>1{print $3}')" != "$LATEST_APP_VERSION_LABEL" ]; do
+            ((COUNTER += 1))
             if [ $COUNTER -gt 120 ]; then
               echo "Timed out waiting for app to be installed with latest version label: $LATEST_APP_VERSION_LABEL"
-              ./bin/kots get apps --namespace $APP_SLUG
+              ./bin/kots get apps --namespace "$APP_SLUG"
               echo "kotsadm logs:"
-              kubectl logs -l app=kotsadm --tail=100 --namespace $APP_SLUG
-              exit -1
+              kubectl logs -l app=kotsadm --tail=100 --namespace "$APP_SLUG"
+              exit 255
             fi
             sleep 1
           done
 
-          printf "App is installed successfully with the correct version label: $LATEST_APP_VERSION_LABEL\n\n"
-          ./bin/kots get apps --namespace $APP_SLUG
+          printf "App is installed successfully with the correct version label: %s\n\n" "$LATEST_APP_VERSION_LABEL"
+          ./bin/kots get apps --namespace "$APP_SLUG"
 
       - name: Generate support bundle on failure
         if: failure()
         uses: ./.github/actions/generate-support-bundle
         with:
-          kots-namespace: "$APP_SLUG"
+          # REVIEW: input `kots-namespace` does not exist on `generate-support-bundle` action
```

Comment:
It does exist here: https://github.com/replicatedhq/kots/blob/main/.github/actions/generate-support-bundle/action.yml#L22

We should add it to the list of input parameters.

Reply:
Is there a sane default?

Reply:
I don't think so, let's make it required I guess?